contrib/mariadb-connector-c-cmake/CMakeLists.txt

```diff
@@ -227,6 +231,15 @@ ${CC_SOURCE_DIR}/libmariadb/secure/openssl_crypt.c
 ${CC_BINARY_DIR}/libmariadb/ma_client_plugin.c
 )
 
+if(WITH_ICONV)
+  find_package(Iconv REQUIRED)
```

Comment:
We don't allow `find_package` in our CMake files.

Reply:
We have strict guidelines for our code base:
- don't use source codes or libraries from OS or repositories;
- build everything from source code;
- use fixed versions of the libraries, no automatic updates;
- all source code of third-party dependencies is checked out;
- all the code can be built with sanitizers and hardening;
- no known memory safety issues exist;
- reproducible (byte-identical), hermetic builds;
- all the code can be cross-compiled;
- minimize the supply chain;
- minimize runtime dependencies on the files on the target machine;
- single-binary distribution;
- static linking;
- no foreign code or plugins can run in the program address space;

Reply:
import libiconv as thirdparty lib for Apple?  add APPLE condition check to find_package Iconv simply?

Reply:
No libraries can be imported, at all, not even on APPLE platform.